composer.json

```diff
@@ -7,15 +7,20 @@
     "license": "Apache-2.0",
     "require": {
         "php": ">=5.4",
-        "google/auth": "dev-master"
+        "google/auth": "dev-master",
+        "monolog/monolog": "^1.17",
+        "phpseclib/phpseclib": "~2.0"
     },
     "require-dev": {
         "phpunit/phpunit": "3.7.*",
         "squizlabs/php_codesniffer": "~2.3"
     },
     "autoload": {
+        "psr-0": {
+            "Google_": "src/"
+        },
         "classmap": [
-            "src/"
+            "src/Google/Service/"
         ]
```

Comment:
I noticed that library is using Guzzle version 5.x. 5.x is mostly in a maintenance mode at this point, and most development efforts are focused on version 6. Lots of libraries are adopting version 6 because it uses PSR-7 and has a much better story for async requests (for example, Drupal 8, Goutte, AWS SDK for PHP, etc.). I would strongly recommend that if you are building some new, that you use Guzzle 6 so that you're using something that will be maintained well into the future, and so that your libraries will be compatible with newer versions of other libraries in the ecosystem. Alternatively, you could support both versions, which helps to future proof your project. This is something we did in the AWS SDK for PHP: https://github.com/aws/aws-sdk-php/tree/master/src/Handler